spec/models/presenter/auction_spec.rb

```diff
@@ -5,6 +5,136 @@
   let(:ar_bids_by_amount) { ar_auction.bids.order('amount ASC, created_at ASC') }
   let(:auction) { Presenter::Auction.new(ar_auction) }
   let(:user) { FactoryGirl.create(:user) }
+  
+  describe 'internal bid methods' do
```

Comment:
Oh I hear you, it just is one of those cases where I have a bunch of complementary methods and I feel like making that determination to organize the other way, but right.